storage/reads/array_cursor.go

```diff
@@ -38,131 +51,16 @@ func newWindowAggregateArrayCursor(ctx context.Context, agg *datatypes.Aggregate
 	}
 }
 
-type arrayCursors struct {
```

Comment:
This isn't used at all. OSS uses `multiShardArrayCursors` which is almost identical in structure except for the underlying series cursor.